airbyte-integrations/connectors/source-teradata/src/main/java/io/airbyte/integrations/source/teradata/envclient/TeradataHttpClient.java

```diff
@@ -0,0 +1,137 @@
+package io.airbyte.integrations.source.teradata.envclient;
```

Comment:
Refactoring question - can you move `envclient` foldet under `test-integration`? Looks like `TeradataHttpClient.java` and other classes in envclient are only used during integration tests.

Reply:
The thing is that the TeradataHttpClient is also used in `test` so if i move it to `test-integration` I will no longer be able to use it there.